src/blocks/section.v2.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: http://json-schema.org/draft-04/schema#
```

Comment:
are we making new versions of all these blocks because `asset-file` is used in `figure` and eventually in these content types?

Reply:
Yep. 😒 

(Sharing the same structure across all the content types is great for consumption as they're all the same. And then painful for adding new/changing existing types...)

Reply:
Good old http://wiki.c2.com/?StableDependenciesPrinciple